Ginger/GingerCoreNET/RecordingLib/RecordingManager.cs

```diff
@@ -273,7 +273,7 @@ private void ElementRecordedHandler(ElementActionCongifuration args)
                 {
                     einfo = (ElementInfo)args.LearnedElementInfo;
                     args.AddPOMToAction = CreatePOM;
-                    args.POMGuid = CurrentPOM.Guid.ToString();
+                    args.POMGuid = CurrentPOM != null ? CurrentPOM.Guid.ToString() : string.Empty;
```

Comment:
as disscused, needs to:
1- check how recording-POM integration working when there isn't any POM selected and how when there are multi POM selections
2- recommend on how it should behave in the difreant cases